### DISCUSSION TITLE
Executing a task or event list when all extensions have been loaded

### DISCUSSION THREAD

**Discussion by Braayy** [Minestom/Minestom#1074] [Help]
Is there a way of creating a task or adding a event listener to a event when it is called when all extensions have been loaded?

**Comment by emortaldev** [comment]
What is the use case? It might be that you only need to add the extension to its dependencies.

**Reply by Braayy** [reply]
I'm doing a system in a custom minestom implementation that generates the resource pack for all registered models, but the models are registered in the extensions. I'm currently re-creating the resource pack every time a extension registers a model
